tests/sql_mode/data/db2.increment.sql

```diff
@@ -0,0 +1,24 @@
+set @@session.sql_mode='ONLY_FULL_GROUP_BY,NO_UNSIGNED_SUBTRACTION,NO_DIR_IN_CREATE,STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ALLOW_INVALID_DATES,ERROR_FOR_DIVISION_BY_ZERO,HIGH_NOT_PRECEDENCE,NO_ENGINE_SUBSTITUTION,REAL_AS_FLOAT';
+-- NO_AUTO_CREATE_USER set failed in mysql8.0
+
+use sql_mode;
+
+-- test sql_mode PIPES_AS_CONCAT
+set @@session.sql_mode=concat(@@session.sql_mode, ',PIPES_AS_CONCAT');
```

Comment:
What is the final effect of the sql_mode since it contains items that are conflict with each other 🤔？ And I think maybe we needn't do this concat

Reply:
The concated sql_mode dose not conflict with each other. The follow is also. So it does not matter whether it is concated ​​or directly assigned.

Reply:
ok